### ISSUE TITLE
Auto silent auth with no static variables?

### ISSUE THREAD

**Issue by Geczy** [auth0-samples/auth0-react-samples#46]
What do you think about using `window.location.origin` instead of static URL?

```js
    <script>
        parent.postMessage({
          hash: window.location.hash,
          type: 'auth0:silent-authentication'
        }, window.location.origin);
    </script>
```

Unsure if this presents any security issues is all 🔒

**Comment by Geczy** [comment]
This won't work in the example since it's on a different server (:3002). But that server isn't necessary, I can make a PR to remove it too

**Comment by Geczy** [comment]
`    <script src="https://cdn.auth0.com/js/auth0/8.8/auth0.min.js"></script>` at the top can also be removed

**Comment by chenkie** [comment]
That would be a good addition, thanks. If you'd like to send in a PR please do.

The example app should be served on 3000. Are you getting 3002?

**Comment by andreasgrimm** [comment]
@Geczy @chenkie at least the removal of the dependency to `auth0.js` in `silent.html` and some config overhead gets addressed in #64

I might be able to send in a PR regarding the other concerns.

I also serve the main app as well as `silent.html` from the same server on a single port. If it's really beneficial in SSO scenarios with more than only one app like mentioned in https://github.com/auth0-samples/auth0-angular-samples/issues/36 then should we make these alternative approaches explicit within the README?
As I'm about to test this anyway I'll provide feedback on this.

Although we use `window.location.origin` instead of a constant value in production, I'm not sure regarding potentially lowered security as well. What do you think?

**Comment by luisrudge** [comment]
We're moving away from the `renewAuth` approach and using `checkSession` instead. No need to have the silent.html anymore
